wasm2c/examples/rot13/main.c

```diff
@@ -45,6 +45,11 @@ void Z_hostZ_buf_done(struct Z_host_instance_t* instance, u32 ptr, u32 size) {
 }
 
 int main(int argc, char** argv) {
+  /* No command-line arguments expected. */
+  if (argc > 1) {
```

Comment:
I think the test and error message here is backwards -- the program runs rot13 on its command-line arguments. So I think we probably want something like `if (argc <= 1)` for this test, and then the usage message would tell the user they're expected to supply at least one word that gets rot13'd?

Reply:
Ah, my bad. Fixed!